### PR TITLE
Update contributing

### PR DESCRIPTION
**Pull request by Mackiovello**
For Starcounter/Guidelines#17

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,40 +1,30 @@
...
-1. Open your terminal/command prompt
-2. Navigate to Cashier/src/Cashier where the `.exe` file is after your build
-3. Run starpack -p 
-4. It should say `succeeded` and point towards the newly created ZIP file
 
-### Upload to Warehouse
-
-1. Follow the instructions in [RebelLounge](https://github.com/Starcounter/RebelsLounge/blob/master/apps_ecosystem/releasing_apps_to_the_warehouse.md) to upload to the warehouses
 
 Cashier.png is made by Freepik from www.flaticon.com
```
Comment (warpech) [inline CONTRIBUTING.md]: Maybe move the above line in README.md?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+Simple Point of Sale application
```
Comment (warpech) [inline README.md]: Add some heading, e.g. `# Cashier`

File: CONTRIBUTING.md (hunk 1 above)
Comment (Mackiovello) [inline CONTRIBUTING.md]: Fixed

File: README.md (hunk 2 above)
Comment (Mackiovello) [inline README.md]: Fixed

**Comment by Mackiovello** [comment]
Points by @warpech are addressed in 	e49cd51

Merging

**Pull request merged**